### PR TITLE
Add jeremyharris/cakephp-lazyload

### PR DESCRIPTION
**Pull request by jeremyharris**
Suggested here: https://github.com/jeremyharris/cakephp-lazyload/issues/3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -246,6 +246,7 @@ Additional lists you might find useful:
 - [PersistRelatedData plugin](https://github.com/riesenia/persist-related-data) - Behavior for persisting selected fields of related models.
 - [RowLocker plugin](https://github.com/lorenzo/row-locker) - Exclusive locks for rows in your tables.
 - [Webservices ORM plugin](https://github.com/usemuffin/webservice) - An ORM like interface for webservices.
+- [JeremyHarris/LazyLoad plugin](https://github.com/jeremyharris/cakephp-lazyload) - An association lazy loader for entities
```
Comment (dereuromark) [inline README.md]: Full stop missing at end, but otherwise looks good.

**Comment by bravo-kernel** [comment]
Not alphabetic ?

**Comment by dereuromark** [comment]
Ops, missed that one. Good point.

**Comment by jeremyharris** [comment]
Wow didn't even notice it was in alphabetical order hah. Whoops, I'll update.

**Comment by makallio85** [comment]
:+1:

**Pull request merged**